docs/developer/best-practices/typescript.asciidoc

```diff
@@ -19,7 +19,7 @@ More details are available in the https://www.typescriptlang.org/docs/handbook/p
 ==== Caveats
 This architecture imposes several limitations to which we must comply:
 
-- Projects cannot have circular dependencies. Even though the Kibana platform doesn't support circular dependencies between Kibana plugins, TypeScript (and ES6 modules) does allow circular imports between files. So in theory, you may face a problem when migrating to the TS project references and you will have to resolve this circular dependency. https://github.com/elastic/kibana/issues/78162 is going to provide a tool to find such problem places.
+- Projects cannot have circular dependencies. Even though the Kibana platform doesn't support circular dependencies between Kibana plugins, TypeScript (and ES6 modules) does allow circular imports between files. So in theory, you may face a problem when migrating to the TS project references and you will have to resolve this circular dependency. With https://github.com/elastic/kibana/issues/78162 we've built a tool to find such problems. Please read the prerequisites section below to know how to use it.
```

Comment:
Can we reference the tool isn't of the PR? Also, do we need to make changes to it for it to still be useful for these migrations? They will have to both references the whitelist and the output to know what is outstanding to fix before migrating to project references.

Reply:
I've updated some wording in the docs I've added. Regarding your question, I believe they just need to make sure they don't have any new circular deps added running our tool and also make sure the plugins they depend on have already migrated into project refs which will also make sure the plugins they rely on are already circular deps free. In that case I don't think they will need to check the allowed list, am I missing something @tylersmalley ?